Extensions/mutualchecker.js

```diff
@@ -0,0 +1,103 @@
+//* TITLE Mutual Checker **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION Adds an icon next to usernames to indicate if they follow you. **//
+//* DETAILS Does what it says on the tin! Made from fresh, distilled Profiler juice. **//
+//* DEVELOPER AprilSylph **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.mutualchecker = new Object({
+	
+	running: false,
+	
+	preferences: {
+		"main_blog": {
+			text: "Blog to check for follow-backs on:",
+			default: "",
+			value: "",
+			type: "blog"
+		},
+		"live_update": {
+			text: "Let me see users follow or unfollow me as I scroll",
+			default: false,
+			value: false,
+			slow: true
+		}
+	},
+	
+	run: function() {
+		this.running = true;
+		if (this.preferences.main_blog.value === "") { this.preferences.main_blog.value = XKit.tools.get_blogs()[0]; }
+		window.xkitmc = new Array();
```

Comment:
Oh right, it's because within $(posts).each "this" means something else, but I forgot I could just call the variable more directly within the loop. Applying changes now